### PR TITLE
Validate in app (just validation)

### PR DESCRIPTION
**Pull request by DennisHouterman**
Still a work in progress, but the app can validate proofs from paper wallet by scanning its QR-code from the scan screen. request to master for demo purposes

### CODE REVIEW

File: validator-frontend/screens/ScanScreen.js [hunk 1]
```diff
@@ -33,7 +52,109 @@ export default class ScanScreen extends React.Component {
   }
 
   handleBarCodeScanned = ({ type, data }) => {
-    uncompressedData = pako.inflate(data, { 'to': 'string' })
-    alert(`Bar code with type ${type} and data ${uncompressedData} has been scanned!`);
+    this._storeData(data)
```
Comment (pimotte) [inline validator-frontend/screens/ScanScreen.js]: We store the data before validation is done, that seems kind of fishy.

File: validator-frontend/screens/ScanScreen.js [hunk 2]
```diff
@@ -33,7 +52,109 @@ export default class ScanScreen extends React.Component {
...
+      '    ' ;
+    let documentJson = JSON.parse(qrString);
+    let claimWithLink = Object.values(documentJson)[0][0];
+    let claimData = Object.values(claimWithLink)[0];
+    let tempdata = claimData[0];
+    let moredata = Object.values(tempdata)[0];
+    console.log(moredata);
+    paperWallet.getCore().registerConnector('ephemeral', new EphemeralConnector())
+    let attestorSsid = await (await paperWallet.getCore().getConnector('ephemeral')).newIdentity({'cert': cert})
+    let validatorSsid = await paperWallet.getCore().newSsid('ephemeral')
```
Comment (pimotte) [inline validator-frontend/screens/ScanScreen.js]: The way we use the paperWallet, we do not actually need to pass a validatorDid. This is only needed if you then want to read the data from the ephemeral connector. We actually just use the data we extracted from the QR.

I would not generate the validatorSsid, and just remove that argument in the validate call

File: validator-frontend/screens/WalletScreen.js [hunk 3]
```diff
@@ -39,19 +44,56 @@ class UitrekselScreen extends Component {
...
+      console.log(error)
+    }
+  }
+
+  _readData = async () => {
+    let displayData = await this._retrieveData();
+    let documentJson = JSON.parse(displayData);
+    let claimWithLink = Object.values(documentJson)[0][0];
+    let claimData = Object.values(claimWithLink)[0];
+    let tempdata = claimData[0];
```
[the remark this answers is not in the document]
Comment (DennisHouterman) [reply validator-frontend/screens/WalletScreen.js]: This seems to be code that has been moved to the wallet. These lines of code and variables will be removed in the next commit

File: validator-frontend/screens/ScanScreen.js (hunk 1 above)
Comment (DennisHouterman) [reply validator-frontend/screens/ScanScreen.js]: This will be moved to the validating screen in case the qr has been verified

**Pull request merged**